packages/playground/src/lib/ai-ui/messages/__tests__/user-messages.test.tsx

```
@@ -0,0 +1,62 @@
+// @vitest-environment jsdom
+import type { MastraDBMessage } from '@mastra/core/agent/message-list';
+import { AssistantRuntimeProvider, ThreadPrimitive, useExternalStoreRuntime } from '@assistant-ui/react';
```

ESLint

[error] `@assistant-ui/react` import should occur before type import of `@mastra/core/agent/message-list`

(import/order)